### PR TITLE
feat: Increase max number of works per artist for new for you GRO-1297

### PR DESCRIPTION
**Pull request by jonallured**
This is a sibling PR to https://github.com/artsy/force/pull/10983 and updates Eigen in the same way for the same reasons.

https://artsyproduct.atlassian.net/browse/GRO-1297

/cc @artsy/grow-devs @shaq31415926 @isakelaris

### CODE REVIEW

File: src/app/Scenes/Home/Components/NewWorksForYouRail.tsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ export const NewWorksForYouRail: React.FC<NewWorksForYouRailProps & RailScrollPr
 
 const artworksFragment = graphql`
   fragment NewWorksForYouRail_artworkConnection on Viewer {
-    artworksForUser(includeBackfill: true, first: 20) {
+    artworksForUser(maxWorksPerArtist: 3, includeBackfill: true, first: 20) {
```
Comment (shaq31415926) [inline src/app/Scenes/Home/Components/NewWorksForYouRail.tsx:77]: Just for my clarification, what does this first: 20 do?

Comment (jonallured) [reply src/app/Scenes/Home/Components/NewWorksForYouRail.tsx:77]: There's a set of arguments in GraphQL that deal with pagination and this is one of them. Used on it's own it's simply saying "give me the first 20" but you can get fancier with cursors and page through a connection of items. More here in the docs!

https://graphql.org/learn/pagination/

Comment (shaq31415926) [reply src/app/Scenes/Home/Components/NewWorksForYouRail.tsx:77]: aah ok, thanks for the explanation. Just for my reference, why do we limit to the first 20?

Comment (shaq31415926) [reply src/app/Scenes/Home/Components/NewWorksForYouRail.tsx:77]: Could we increase this please? As we show more artworks per artist, we are potentially capping relevant artworks for the user?

Comment (jonallured) [reply src/app/Scenes/Home/Components/NewWorksForYouRail.tsx:77]: We just need to pick a number! Feel free to suggest a new page size and I can update it here!

Comment (shaq31415926) [reply src/app/Scenes/Home/Components/NewWorksForYouRail.tsx:77]: I am not sure why we are capping it. I would say 100?

Comment (jonallured) [reply src/app/Scenes/Home/Components/NewWorksForYouRail.tsx:77]: If a user does not have that many then backfill kicks in and then if there still isn't enough then it's just a limit and it'll show less. Does that make sense?

Comment (shaq31415926) [reply src/app/Scenes/Home/Components/NewWorksForYouRail.tsx:77]: Lets go with 100 then?!

Comment (jonallured) [reply src/app/Scenes/Home/Components/NewWorksForYouRail.tsx]: Done!

Comment (jonallured) [reply src/app/Scenes/Home/Components/NewWorksForYouRail.tsx]: Oh damn I'm totally wrong - this is for the rail on the home screen - 20 is very reasonable here! 😬 Going to go back to 20.

Comment (shaq31415926) [reply src/app/Scenes/Home/Components/NewWorksForYouRail.tsx:77]: aah ok, that makes sense haha.

File: src/app/Scenes/NewWorksForYou/NewWorksForYou.tsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ import { Box, SimpleMessage, Spacer } from "palette"
 import { createPaginationContainer, graphql, QueryRenderer, RelayPaginationProp } from "react-relay"
 
 const SCREEN_TITLE = "New Works for You"
-const PAGE_SIZE = 20
+const PAGE_SIZE = 100
```
[the remark this answers is not in the document]
Comment (jonallured) [reply src/app/Scenes/NewWorksForYou/NewWorksForYou.tsx:15]: Good catch - I've updated the count in the other place so this should work now. I'll verify in a beta once this is merged!

**Review by pam- — APPROVED** [review]

**Pull request merged**